packages/esm-patient-registration-app/src/offline.resources.ts

```diff
@@ -7,16 +7,16 @@ import { PatientIdentifierType, FetchedPatientIdentifierType } from './patient-r
 import { cacheForOfflineHeaders } from './constants';
 
 export interface Resources {
-  addressTemplate: any;
+  addressTemplate: undefined | any;
```

Comment:
This seems like a weird change? Surely `undefined` is covered by `any` and, in any case, could be represented by `addressTemplate?: any`